### ISSUE TITLE
Either static libraries or dynamic frameworks? Or ... static frameworks?

### ISSUE THREAD

**Issue by claybridges** [CocoaPods/blog.cocoapods.org#158]
[Here](http://blog.cocoapods.org/CocoaPods-1.7.0-beta/), @dnkoutso writes:

> Currently, consumers of pods are limited to an all-or-nothing approach to integrate their project by forcing them to make a choice on whether to use dynamic frameworks everywhere or static libraries everywhere via the use_frameworks! option.

But if the source Podspec specifies a static framework (e.g. `s.static_framework = true
`, isn't that what the consumer gets, even with a `use_frameworks!` declaration?

**Comment by dnkoutso** [comment]
More or less both statements are correct. I do not see an issue here so I am going to close and we can continue discussion here. Especially for a 1.8.x release.

**Comment by paulb777** [comment]
`s.static_framework = true` means if the `use_frameworks!` declaration is set in the Podfile, the Pod should be built as a static framework. `s.static_framework = true` has no impact if `use_frameworks!` is not in the Podfile.

**Comment by claybridges** [comment]
Right, but the choice is between frameworks and libraries, not between "dynamic frameworks" and "static libraries". Whether the framework is static or dynamic is not strictly determined by the `use_frameworks!` declaration.
